harness/determined/core/_context.py

```diff
@@ -359,6 +390,7 @@ def init(
         _metrics=metrics,
         _tensorboard_manager=tensorboard_manager,
         _session=session,
+        info=info,
```

Comment:
Just for my understanding, what does `info` do?

Reply:
Because we want to access the experiment id and trial id in `alert` function